src/lib.rs

```diff
@@ -6,6 +6,7 @@ use std::net::SocketAddr;
 use thiserror::Error;
 
 pub mod calendar;
+mod google_calendar;
```

Comment:
I'd probably put the module at `src/calendar/google/mod.rs` since it conceptually belongs to the `calendar` module.